Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -748,7 +748,9 @@ - (UITableViewCell *)cellForArchivedConversationsRow:(UITableView *)tableView
     // Constrain to cell margins.
     [stackView autoPinEdgeToSuperviewMargin:ALEdgeLeading relation:NSLayoutRelationGreaterThanOrEqual];
     [stackView autoPinEdgeToSuperviewMargin:ALEdgeTrailing relation:NSLayoutRelationGreaterThanOrEqual];
-    // NOTE: It's critical that we pin to the superview top and bottom _edge_ and not _margin_.
+    // Ensure that the cell's contents never overflow the cell bounds.
+    // We pin pin to the superview _edge_ and not _margin_ for the purposes
+    // of overflow, so that changes to the margins do not trip these safe guards.
     [stackView autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:0 relation:NSLayoutRelationGreaterThanOrEqual];
```

Comment:
> // We pin pin to the superview _edge_ and not _margin_ for the purposes  
> // of overflow, so that changes to the margins do not trip these safe guards.

nit: typo (pin pin)

If we *did* add margins, the current implementation would allow the view that we're trying to constraint to overflow past the top/bottom margins, which seems undesirable. Pinning to the margins will make the view always at least as big as you have it now, but additionally enforce any desired margin from the edge.

So to me it still seems like pinning to edges here is worse than pinning to margins.

Admittedly this is all hypothetical with a 0 margin as you have here, but this code has now been duplicated into a second place, and presumably you'll continue to use it, so I'm trying to either:

1. convince you that this is wrong and you should stop using it.
- or - 
2. come to understand why this is better and start using this pattern myself.